### DISCUSSION TITLE
Any way to have different game over song play on a different map?

### DISCUSSION THREAD

**Discussion by ToonLink0001** [nzp-team/nzportable#877] [General]
Hey I was wondering if there is a way to change the game over song on different maps like Warehouse has one that is different to Nacht, it there a way to do that or do they all have to have the same one?

**Comment by MotoLegacy** [comment]
currently you're stuck with the stock one, but soon-ish the game will allow you to provide custom paths per-map for all of the music and the power-ups etc.

**Reply by ToonLink0001** [reply]
ok thanks, I'll look forward to that!
